static/js/board_carousel.js

```diff
@@ -149,33 +152,54 @@ function deleteElement(bobid){
   if ($('#slideshow').hasClass('initialized')) {
     $('#slideshow').removeClass('initialized');
   }
-
   // reinit the carousel
   $('#slideshow').carousel({
-    fullWidth: true
+    fullWidth: true,
+    onCycleTo: function(activeItem){
+      updateVoteLabel(activeItem);
+      loadVideo(activeItem);
+    }
```

Comment:
it looks like we do this a few times, should we just make a `reInit(){}` function?